APMrover2/sailboat.cpp

```diff
@@ -75,6 +75,42 @@ const AP_Param::GroupInfo Sailboat::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("NO_GO_ANGLE", 6, Sailboat, sail_no_go, 45),
 
+    // @Param: SAIL_MAX_X_TRACK
+    // @DisplayName: Sailing vehicle max cross track error
+    // @Description: The sail boat will tack when it reaches this cross track error, defines a corridor of 2 times this value wide, 0 disables
+    // @Units: m
+    // @Range: 5 25
+    // @Increment: 1
+    // @User: Standard
+    AP_GROUPINFO("MAX_X_TRACK", 7, Sailboat, max_cross_track, 10),
+
+    // @Param: SAIL_LOIT_RADIUS
+    // @DisplayName: Loiter radius
+    // @Description: Vehicle will sail when within this distance of the target position
+    // @Units: m
+    // @Range: 0 20
+    // @Increment: 1
+    // @User: Standard
+    AP_GROUPINFO("LOIT_RADIUS", 8, Sailboat, loit_radius, 5),
+
+    // @Param: SAIL_OPTIONS
+    // @DisplayName: Sailboat options bitmask
+    // @Description: Sailboat options bitmask
+    // @Values: 0:None,1:Aux Sail,2:Aux Throttle Limit
+    // @Bitmask: 0:Aux throttle main for sail,1:Aux Throttle Limit
+    // @Increment: 1
+    // @User: Standard
+    AP_GROUPINFO("OPTIONS", 9, Sailboat, sail_options, 0),
+
+    // @Param: SAIL_MIN_WNDSPD
+    // @DisplayName: Minimum wind speed to sail in
+    // @Description: Sailboat minimum wind speed to continue sail in, at lower wind speeds the sailboat will motor if one is fitted
+    // @Units: m/s
+    // @Range: 0 5
+    // @Increment: 1
+    // @User: Standard
+    AP_GROUPINFO("MIN_WNDSPD", 10, Sailboat, sail_assist_windspeed, 0),
```

Comment:
normally I prefer if we can put the MIN a the end of the parameter name..

Reply:
fixed